lib/media/track/localaudiotrack.js

```diff
@@ -173,4 +176,44 @@ class LocalAudioTrack extends LocalMediaAudioTrack {
  * @event LocalAudioTrack#stopped
  */
 
+/**
+ * A {@link NoiseCancellation} allows you to control enable/disable noise cancellation at runtime.
+ * This interface is exposed on {@link LocalAudioTrack} property `noiseCancellation`. It is available only when
+ * {@link NoiseCancellationOptions} are specified when creating a {@link LocalAudioTrack}
+ * @typedef {object} NoiseCancellation
+ * @property {NoiseCancellationVendor} vendor - identifies the noise cancellation library vendor.
+ * @property {function} enable - enables noise cancellation.
+ * @property {function} disable - disables noise cancellation.
```

Comment:
I wonder if it make sense to make `NoiseCancellation` a class so you can properly document these two functions - with information about params and return types.

Reply:
good Idea, I have documented this as an `interface`.